.github/workflows/centos7-ci.yml

```diff
@@ -2,11 +2,9 @@ name: CI Centos7
 
 on:
   push:
-    branches:
-      - master
+    branches: [master, 'release/**']
   pull_request:
-    branches:
-      - master
+    branches: [master]
```

Comment:
In fact, we will create PR to `release/**` right?

Reply:
Yes and relevant discussion is in #3726. Welcome have some discussion~

Reply:
BTW, your change request still blocks the merging 🤣